### ISSUE TITLE
Use of 7th chords in Key.chords()

### ISSUE THREAD

**Issue by Maarius** [danigb/tonal#61]
The output of `Key.chords("C major")` is `CMaj7, Dm7, Em7, FMaj7, G7, Am7, Bm7b5`. 

Why are the 7th chords used? Shouldn't it be `C, Dm, Em, F, G, Am, Bdim` like for instance here https://www.theguitarlesson.com/guitar-theory/guitar-chords/key-c-major/?

**Comment by rickdps** [comment]
Yes. That's exactly what I need. It would be nice to have a way to choose the number of notes a chord has.

Any progress on this features? 

Thank you all. It is a great library indeed.

**Comment by danigb** [comment]
Hi @Maarius and @rickdps 

I'm unsure of the change, because you can easily get the triads from the 7th chords (and not the opposite). 

What about:  

```
const triadOf7th = name => name.slice(0, name.length -1)
Key.chords("C major").map(triadOf7th)
```

Maybe it's better to add the helper method to the library instead of change the `chords` method... thoughts?

Sorry for the delay and thanks for helping!
